proxy.go

```diff
@@ -53,10 +54,29 @@ func (rp RelengapiProxy) runForever() {
 	// httputil's ReverseProxy is not specifically "reverse", and it will
 	// do fine here.  The director transforms outgoing requests.
 	director := func(req *http.Request) {
-		// point toward the upstream server
-		req.URL.Scheme = "https"
-		req.URL.Host = rp.relengapiHost
-		req.Host = rp.relengapiHost
+        // redirect tooltool traffic to tooltool.mozilla-releng.net
+        if strings.HasPrefix(req.URL.Path, "/tooltool") {
+            req.URL.Scheme = "https"
+            req.URL.Path = strings.TrimPrefix(req.URL.Path, "/tooltool")
+            req.URL.RawPath = strings.TrimPrefix(req.URL.RawPath, "/tooltool")
```

Comment:
Since go 1.5, [EscapedPath()](https://golang.org/pkg/net/url/#URL.EscapedPath) method was added for returning the raw path, and the `RawPath` property is not directly accessed by the internal library.

[Here](https://github.com/golang/go/blob/go1.8.3/src/net/url/url.go#L640-L660) you see that if it is unset, the escaped path will be evaluated, whereas if `RawPath` is actually set, it must first be unescaped and compared to `Path`, and only if successful is that returned, otherwise `Path` is escaped. This is basically to account for there being potentially several `RawPath`s that can evaluate to the same `Path`, and setting it to one of them allows that to be the favoured raw path. In our case, we don't care, so I'd suggest changing to simply:

```
req.Url.RawPath = ""
```